sub-projects/device/src/aws.py

```diff
@@ -0,0 +1,42 @@
+import boto3
+import json
+from PIL import Image
+
+class Rekgoniton(object):
```

Comment:
I think this stuff is still double tabbed

Reply:
Also `Rekgonition` -> `Rekognition`

Reply:
Yea I just rewrote the file to fix that. Class is now AWS.